stack.yaml.lock

```diff
@@ -0,0 +1,12 @@
+# This file was autogenerated by Stack.
+# You should not edit this file by hand.
+# For more information, please see the documentation at:
+#   https://docs.haskellstack.org/en/stable/lock_files
+
+packages: []
+snapshots:
+- completed:
+    size: 515546
+    url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/8/17.yaml
+    sha256: d049868149cb8be5387b225f8994c28ba1ce1887a34fc67c6d6769c899976702
+  original: lts-8.17
```

Comment:
Just like you suggested, it'd be good to update the lts described on the `stack.yaml` file

Reply:
Also, I will update the module documentation is wrong